code/modules/status_system/statusEffects.dm

```diff
@@ -1279,17 +1279,20 @@
 		var/const/max_health = 30
 		var/const/max_stam = 60
 		var/const/regen_stam = 5
-		var/const/max_dist = 50
 		var/mob/living/carbon/human/H
 		var/datum/gang/gang
 		var/on_turf = 0
 
+		var/elapsedTime = 0
```

Comment:
Could just use `ON_COOLDOWN` instead

Reply:
this accounts for mult atm. is there an alternative that does this?

Reply:
`ON_COOLDOWN` uses `timeofday` so should be adjusted anyway, it's not dependent on server tick rate (I think)